### PR TITLE
Update nodejs version on readme instructions

### PR DESCRIPTION
**Pull request by oliveiraleo**
Hello,

When building webconsole from scratch using nodejs packed by Ubuntu's official package repositories, I ran into the same issue as the one described here: https://github.com/free5gc/webconsole/issues/25

When trying to follow the instructions on the README file, I received a deprecation warning from nodejs script. Then, according to [this document here](https://github.com/nodesource/distributions#ubuntu-versions), the currently supported versions of nodejs are 18, 19 and 20. From the information of this other [issue here](https://github.com/free5gc/free5gc/issues/484) on the free5gc main repo by me, I chose to install version 18.x and then webconsole built successfully.

This PR will update the instructions on the readme file on this repo too

PS: ASAP I'll update the instructions on the free5gc guide so we have the most up to date information in both places

TIA

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -5,16 +5,17 @@
 sudo apt remove cmdtest yarn
 curl -sS https://dl.yarnpkg.com/debian/pubkey.gpg | sudo apt-key add -
 echo "deb https://dl.yarnpkg.com/debian/ stable main" | sudo tee /etc/apt/sources.list.d/yarn.list
```
Comment (yccodr) [inline readme.md:7]: Since the corepack is installed by default, we can just `corepack enable` to setup Yarn.

```suggestion
```

File: readme.md [hunk 2]
```diff
@@ -5,16 +5,17 @@
...
-sudo apt-get update
-sudo apt-get install -y nodejs yarn
+sudo apt update
+sudo apt install -y yarn
```
Comment (yccodr) [inline readme.md:9]: Same reason as mentioned aboved

```suggestion
```

File: readme.md [hunk 3]
```diff
@@ -5,16 +5,17 @@
...
 ```
 
 ### Install Node.js
 ```bash
 sudo apt remove nodejs -y
-curl -s https://deb.nodesource.com/setup_16.x | sudo bash
+curl -fsSL https://deb.nodesource.com/setup_18.x | sudo -E bash -
```
Comment (yccodr) [inline readme.md:15]: The Node.js 20 is the latest LTS version and works fine with this repo.

```suggestion
curl -fsSL https://deb.nodesource.com/setup_20.x | sudo -E bash -
```

**Review by andy89923 — APPROVED** [review]
LGTM

**Comment by oliveiraleo** [comment]
While testing free5gc, I found that root permission is necessary in order to allow corepack to correctly setup its environment on `/usr/bin` folder. And since the mentioned folder is writable by root only, then the permission becomes a requirement (first screenshot attached below). 

image

To fix this behavior, it's only a matter of appending `sudo` to the command so corepack can escalate privileges (see https://github.com/free5gc/webconsole/pull/71/commits/fbfe3f5b48a5a39a196dc6fc34d7954ba804ac49 and the screenshot below).

image

Regards, Leo.

**Review by tim-ywliu — APPROVED** [review]

**Pull request merged**